nodejs/awsx/apigateway/api.ts

```diff
@@ -283,12 +288,27 @@ export interface Endpoint {
 
 export interface APIArgs {
     /**
-     * Routes to use to initialize the APIGateway.
+     * Routes to use to initialize the APIGateway.  These will be used to create the Swagger
+     * specification for the API.
      *
-     * Either [swaggerString] or [routes] must be specified.
+     * Either [swaggerString] or [routes] or [additionalRoutes] must be specified.  [routes] can be
+     * provided along with [additionalRoutes].
      */
     routes?: Route[];
 
+    /**
+     * Routes to use to initialize the APIGateway.  These will be used to create the Swagger
+     * specification for the API.
+     *
+     * Unlike `routes`, these can be provided in `Input` form (i.e. `pulumi.Output` are allowed).
```

Comment:
Can you add some use cases to this comment? Its not clear to my why I would want to use this.

Reply:
Sure.